### ISSUE TITLE
Improve search results relevancy

### ISSUE THREAD

**Issue by gerard-kanters** [arckinteractive/elgg_solr#12]
This is more of a discussion item at the moment. 

This document is a good start to read: https://wiki.apache.org/solr/SolrRelevancyFAQ 

Interesting topic is param PS (phrase slop) where you can influence relevancy and search ranking if the searched words are in close proximity of each other. 

This could be a plugin setting to play with. I think a low value for PS (<10) is preferred since most users search for a string in close proximity and two words that are far between might not be relevant at all.

Other interesting doc to read is how default behavior is and how to influence it:

https://lucene.apache.org/core/4_0_0/core/org/apache/lucene/search/similarities/TFIDFSimilarity.html

In a nutshell, SOLR uses TF IDF to determine relevancy. TF stands for Term Frequency, the more terms occur in a doc the higher it is get scored. IDF stands for  inverse document frequency where common terms in a search query like "the" are less valued than rarer terms.
